test/functional/validator-options.spec.ts

```diff
@@ -44,4 +44,9 @@ describe('validator options', () => {
       expect(errors.length).toEqual(0);
     });
   });
+
+  it(`should forbidUnknownValues by default`, function () {
+    expect(validator.validateSync({}).length).toEqual(1);
+    expect(validator.validateSync({}, {}).length).toEqual(1);
+  });
```

Comment:
```suggestion
  it(`should returns error on unknown objects if options object argument is passed`, function () {
    const anonymousObject = { badKey: 'This should not pass.' }
    expect(validator.validateSync(anonymousObject, undefined).length).toEqual(1);
    expect(validator.validateSync(anonymousObject, {}).length).toEqual(1);
  });
```

I would suggest that you check something other than just the length. Perhaps do a check for `errors[0].constraints` in addition?

Reply:
Cool. I just updated that file further based on your feedback and to try to match the style of the rest of the file.